db/seeds.rb

```diff
@@ -736,3 +736,9 @@ def describe_teacher(teacher)
 end
 
 print_seed_info('Adding funding exemptions:')
+
+print_seed_info('Adding APITokens:')
```

Comment:
After Mook merges his pr for providers, we can add a token for each provider, but will wait for that one